goverge/coverage.py

```diff
@@ -20,14 +20,18 @@
 import sys
 import threading
 
+_failed = False
+
 
 def check_failed(return_code):
     """Check if the test run failed and exit with the correct code if it did.
     :type return_code: int
     :param return_code: The return code of a test run
     """
     if return_code != 0:
-        os._exit(return_code)
+        global _failed
+        _failed = True
+        print("This is FAILED's value: {}".format(_failed))
```

Comment:
do you still need this?

Reply:
Nope debugging statement that I clearly forgot to remove....